cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/JmsSource.java

```diff
@@ -176,8 +176,7 @@ private void initializeJMSConnection(Hashtable<String, String> envVars, String d
           LOG.warn("Exception when closing connection", ex2);
         }
       }
-      throw new RuntimeException("JMSException thrown when trying to initialize connection: " + ex.getMessage(),
-                                 ex);
+      throw new RuntimeException("JMSException thrown when trying to initialize connection: " + ex.getMessage(), ex);
```

Comment:
Not your PR, but probably don't need the ``ex.getMessage()`` when you're already logging with the Exception parameter.